.github/workflows/test_ipex.yml

```diff
@@ -65,4 +65,4 @@ jobs:
 
       - name: Test with Pytest
         run: |
-          pytest tests/ipex/${{ matrix.test-file }}
+          pytest tests/ipex/${{ matrix.test-file }} -rsx -v
```

Comment:
what's ''rsx" for ?

Reply:
It will output more detailed logs for pytests so we can easily know which test failed with what reason.